multi_physics/QED/QED_table_generator/table_generator.cpp

```diff
@@ -9,12 +9,15 @@
 #include <vector>
 #include <algorithm>
 #include <fstream>
-#include <omp.h>
 
-#include "../QED/src/physics/quantum_sync/quantum_sync_engine_tables.hpp"
-#include "../QED/src/physics/quantum_sync/quantum_sync_engine_tables_generator.hpp"
-#include "../QED/src/physics/quantum_sync/quantum_sync_engine_tabulated_functions.hpp"
-#include "../QED/src/physics/breit_wheeler/breit_wheeler_engine_tables_generator.hpp"
+#ifdef PXRMP_TABLE_GEN_HAS_OPENMP
+    #include <omp.h>
+#endif
+
+#include <picsar_qed/physics/quantum_sync/quantum_sync_engine_tables.hpp>
+#include <picsar_qed/physics/quantum_sync/quantum_sync_engine_tables_generator.hpp>
+#include <picsar_qed/physics/quantum_sync/quantum_sync_engine_tabulated_functions.hpp>
+#include <picsar_qed/physics/breit_wheeler/breit_wheeler_engine_tables_generator.hpp>
```

Comment:
Uh, the internal includes here should actually use `"..."` to avoid prefering an externally installed, potentially older version of picsar.

```suggestion
#include "picsar_qed/physics/quantum_sync/quantum_sync_engine_tables.hpp"
#include "picsar_qed/physics/quantum_sync/quantum_sync_engine_tables_generator.hpp"
#include "picsar_qed/physics/quantum_sync/quantum_sync_engine_tabulated_functions.hpp"
#include "picsar_qed/physics/breit_wheeler/breit_wheeler_engine_tables_generator.hpp"
```

Reply:
Ah, sure! thanks!